app/views/exams/components/ExamLevel.vue

```diff
@@ -19,10 +19,14 @@ export default {
       type: Number,
       required: true,
     },
+    language: {
+      type: String,
+      required: true,
```

Comment:
failing here
<img width="731" alt="Screenshot 2024-10-21 at 1 04 23 PM" src="https://github.com/user-attachments/assets/baf5a055-3244-4d8b-9bd4-a62475da7564">

Also, I dont see code-language set in database